README.md

```diff
@@ -384,6 +384,17 @@ that you want to force the cache to rebuild.  You can do this by using the
 tables from the cache, re-create the tables, then populate the cache with the
 data that is in the politeiad git repositories.
 
+##### Building with repository version
+
+It is often useful to have version information from the repository where politeia
```

Comment:
80 col limit

Also, include the exact cli command that should be run like in the other sections.  Make it easy for people.

Reply:
Fixed :+1: